include/rocksdb/statistics.h

```diff
@@ -396,7 +396,7 @@ enum StatsLevel {
   // Collect all stats except the counters requiring to get time inside the
   // mutex lock.
   kExceptTimeForMutex,
-  // Collect all stats expect time inside mutex lock AND time spent on
+  // Collect all stats except time inside mutex lock AND time spent on
   // compression
   kExceptDetailedTimers,
```

Comment:
That was the original idea, but we thought that changing the enum could be problematic since this is a public interface. Changing it would certainly be a cleaner solution, though.